src/pynwb/validate.py

```diff
@@ -31,22 +31,22 @@ def print_errors(errors):
     args = parser.parse_args()
 
     if not os.path.exists(args.path):
-        write('%s not found' % path, file=sys.stderr)  # noqa: F821
+        write_out('%s not found' % args.path)
```

Comment:
@oruebel  This new PR cleans up the call to `write_out` removing extra unused parameters like `file=sys.stderr` and `end=''` that were present in #187 

Is expected to write error on `stdout` instead of `stderr` ? 

Also `sys.out` is invalid, it should be `sys.stdout`

Reply:
Probably stderr.